ibis/tests/all/test_temporal.py

```diff
@@ -41,9 +41,11 @@ def test_timestamp_extract(backend, alltypes, df, attr):
 @pytest.mark.parametrize('unit', [
     'Y', 'M', 'D',
     param('W', marks=pytest.mark.xfail),
-    'h', 'm', 's', 'ms', 'us', 'ns'
+    'h', 'm', 's', 'ms', 'us',
+    param('ns', marks=pytest.mark.xfail)
```

Comment:
Does this pass now that you added the `skipif_backend`?

Reply:
Yes, for some reason when it try to test `ns` unit .. raise an error that the pytest breaks ... 
I tried using pytest mark for `ns` and it doesn't work .. I need to skip that for mapd.
for another tests here .. I could remove `skipif_backend` and just put some xfail for some units and works good.

Reply:
you're right! sorry, I will remove this now.